cypress/e2e/viewAnApplication.cy.ts

```diff
@@ -45,17 +44,17 @@ describe('Application view page', () => {
       cy.task('clearNock');
     });
   });
+
   screenPresets.forEach((screenPreset) => {
     it(`as a manager group user I can edit all application details on ${screenPreset}`, () => {
-      cy.viewport(screenPreset);
-
       cy.task('clearNock');
       cy.clearCookies();
+      cy.viewport(screenPreset);
       cy.loginAsUser('manager');
 
       ApplicationsPage.visit();
       ApplicationsPage.getSearchInput().should('be.visible');
-      ApplicationsPage.getSearchInput().type('1');
+      ApplicationsPage.getSearchInput().type('John');
```

Comment:
Based on the PR description it seems like the PR s tackling some sort of tests flakiness.

This change, however, seems to change the test itself. The before and after do not seem equivalent.
Does this change solve some sort of issue?

Reply:
This specific change is because pipeline test is connected to the dev api, but we're building the mocks/fixtures in a following ticket. 

The reason for this change is that the '1' record on dev api is not complete and in parts has bad data. So this is just a temp fix.